manifests/application.pp

```diff
@@ -15,21 +15,8 @@
   include certs::apache
   include certs::candlepin
   include certs::foreman
-  include certs::pulp_client
```

Comment:
Making a note to fix this, we'll still need to deploy these certificates for N-1 support.